server/etcdmain/help.go

```diff
@@ -107,6 +107,8 @@ Member:
 Clustering:
   --initial-advertise-peer-urls 'http://localhost:2380'
     List of this member's peer URLs to advertise to the rest of the cluster.
+  --set-member-localaddr 'false'
+    Enable to have etcd use the first specified and non-loopback host from initial-advertise-peer-urls as the local address when communicating with a peer.
```

Comment:
Would it make more sense to say `local address` rather than `host`?  Basing this on the test case above where local IPv4 is prioritised over hostname. 

```suggestion
    Enable to have etcd use the first specified and non-loopback local address from initial-advertise-peer-urls as the local address when communicating with a peer.
```

Alternatively we might just want to state what the priority order is so users aren't left to figure this out on their own.